src/commands/project/retrieve/preview.ts

```diff
@@ -36,6 +36,11 @@ export default class RetrieveMetadataPreview extends SfCommand<PreviewResult> {
       summary: messages.getMessage('flags.target-org.summary'),
       required: true,
     }),
+    concise: Flags.boolean({
+      summary: messages.getMessage('flags.concise.summary'),
+      description: messages.getMessage('flags.concise.description'),
```

Comment:
After you update flags.concise.summary with the extra info, you can remove this.